independent-publisher-connectors/OpenWeather/apiDefinition.swagger.json

```diff
@@ -1,222 +0,0 @@
-{
```

Comment:
Hi @thatapiguy, I hope you are doing well. because of the connectors mix in the PR, I suggest you close this PR and resubmit a new PR with the appropriate connector files only.
Please let us know if you have any questions.
Thank you very much.

Reply:
> Hi @thatapiguy, I hope you are doing well. because of the connectors mix in the PR, I suggest you close this PR and resubmit a new PR with the appropriate connector files only. Please let us know if you have any questions. Thank you very much.

I submitted a new PR - https://github.com/microsoft/PowerPlatformConnectors/pull/1565